Tools/message_id.py

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
@dagar You want all the RTPS nodes using the same numbering (even ones previously built). The values should never change.